lib/fs.js

```diff
@@ -3171,15 +3172,11 @@ function glob(pattern, options, callback) {
   callback = makeCallback(callback);
 
   const Glob = lazyGlob();
-  // TODO: Use iterator helpers when available
```

Comment:
Note: this TODO was added in https://github.com/nodejs/node/commit/151d365ad14187d39b912a323a322f900dd0368c and already resolved in https://github.com/nodejs/node/commit/090add7864a130f8ce8774332ec4de00b9c01ca1